.github/workflows/tests.yml

```diff
@@ -14,13 +14,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11.0-rc.2", "pypy3"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11.0-rc.2", "pypy3.7"]
```

Comment:
Can you change 3.11.0-rc.2 to 3.11 so that we don't make another PR for that please?

Reply:
I was about to ask about that haha

Reply:
Done!